tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -44,10 +45,24 @@
 EXAMPLES_KEY = 'examples'
 
 
-def _PartitionFn(record: bytes, num_partitions: int, buckets: List[int]) -> int:
+def _PartitionFn(record: bytes, num_partitions: int, buckets: List[int], 
+                 split_config: example_gen_pb2.SplitConfig) -> int:
   assert num_partitions == len(
       buckets), 'Partitions do not match bucket number.'
-  bucket = int(hashlib.sha256(record).hexdigest(), 16) % buckets[-1]
+  partition_str = record
+  if split_config.HasField("splitting_method_config"):
```

Comment:
still check partition_feature_name not the entire config (as we might add other field in oneof)